src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  width: 100%;
+  grid-auto-flow: row;
+  grid-template-columns: repeat($numberOfColums, 200px);
```

Comment:
There is a typo in the variable name `$numberOfColums`. It should be `$numberOfColumns` to match the intended variable name and avoid errors.